bin/vernemq.sh

```diff
@@ -182,7 +184,7 @@ siguser1_handler() {
 # SIGTERM-handler
 sigterm_handler() {
     if [ $pid -ne 0 ]; then
-        if [ -d "${SECRETS_KUBERNETES_DIR}" -a -f "${TOKEN_FILE}" ] ; then
+        if env | grep "DOCKER_VERNEMQ_DISCOVERY_KUBERNETES" -q; then
```

Comment:
is this really equivalent functionality in deleted and new 'if' clauses?

Reply:
I think users would probably use a modified version of the script anyways if they would run on Kubernetes without enabling `DOCKER_VERNEMQ_DISCOVERY_KUBERNETES`.

Reply:
But if you insist, I will change this line back.

Reply:
Yep, thanks, let's revert this line. small, safe steps (although I'm sure you're likely right with your remarks)

Reply:
fyI: I removed the line.